azure-pipelines.yml

```diff
@@ -1,5 +1,11 @@
 name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd).$(BuildCounter)
 
+parameters:
+  - name: terraformapply
```

Comment:
What does this parameter do? Please can we have parameter name accordingly.

Reply:
Changed param name to ContinueEvenIfResourcesAreGettingDestroyed